cmd.go

```diff
@@ -77,17 +75,21 @@ func (r *Runner) SetupRoot(cmd *cobra.Command) *Runner {
 }
 
 // Setup sets up the Command
-func (r *Runner) Setup(cmd *cobra.Command, runner interface{}, cfg interface{}) *Runner {
+func (r *Runner) Setup(cmd *cobra.Command, runner fx.Option, _ any) *Runner {
 	if cmd.RunE == nil {
-		cmd.RunE = r.runE(runner, cfg)
+		cmd.RunE = r.runE(runner)
 	}
 
 	cmd.AddCommand(&cobra.Command{
 		Use:   "init",
 		Short: "output the config environment variables and exits",
 		Args:  cobra.NoArgs,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			vars, err := GetVariablesFromConfig(r.prefix, cfg)
+			var cfgMgr config.Provider
+			app := fx.New(Module, fx.Populate(&cfgMgr))
```

Comment:
We'll have to include the runner module here too since it is the one that adds all the ConfigDefinitions.